infra/definitions/CoreFx-Linux-Product-Native-Trusted.json

```diff
@@ -0,0 +1,749 @@
+      }
+    },
+    {
+      "enabled": false,
+      "definition": {
+        "id": "57578776-4c22-4526-aeb0-86b6da17ee9c"
+      },
+      "inputs": {
+        "additionalFields": "{}"
+      }
+    }
+  ],
+  "variables": {
+    "system.debug": {
+      "value": "false",
+      "allowOverride": true
+    },
+    "Platform": {
+      "value": "x64",
+      "allowOverride": true
+    },
+    "ConfigurationGroup": {
+      "value": "Release",
+      "allowOverride": true
+    },
+    "GitRepo": {
+      "value": "$(VsoCorefxGitUrl)"
+    },
+    "GitDirectory": {
+      "value": "/root/corefx"
+    },
+    "DockerContainerName": {
+      "value": "$(Build.BuildId)"
+    },
+    "DockerImageName": {
+      "value": "$(DockerRepository):$(DockerTag)"
+    },
+    "DockerRepository": {
+      "value": "chcosta/dotnetcore"
+    },
+    "DockerTag": {
+      "value": "debian82_prereqs_2",
+      "allowOverride": true
+    },
+    "CloudDropAccountName": {
+      "value": "dotnetbuildoutput"
+    },
+    "CloudDropAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "OfficialBuildId": {
+      "value": "$(Build.BuildNumber)",
+      "allowOverride": true
+    },
+    "Label": {
+      "value": "$(Build.BuildNumber)",
+      "allowOverride": true
+    },
+    "SourceVersion": {
+      "value": "HEAD",
+      "allowOverride": true
+    },
+    "RepoUser": {
+      "value": "dn-bot"
+    },
+    "RepoAccessToken": {
+      "value": null,
+      "isSecret": true
+    },
+    "DockerCopyDest": {
+      "value": "$(Build.BinariesDirectory)/docker_repo"
+    },
+    "VsoAccountName": {
+      "value": "dn-bot"
+    },
+    "VsoPassword": {
+      "value": null,
+      "isSecret": true
+    },
+    "VsoRepoName": {
+      "value": "DotNet-CoreFX-Trusted",
+      "allowOverride": true
+    },
+    "VsoCorefxGitUrl": {
+      "value": "https://$(VsoAccountName):$(VsoPassword)@devdiv.visualstudio.com/DevDiv/_git/$(VsoRepoName)/"
+    }
+  },
+  "demands": [
+    "Agent.OS -equals linux"
+  ],
+  "retentionRules": [
+    {
+      "branches": [
+        "+refs/heads/*"
+      ],
+      "artifacts": [
+        "build.SourceLabel"
+      ],
+      "artifactTypesToDelete": [],
+      "daysToKeep": 10,
+      "minimumToKeep": 1,
+      "deleteBuildRecord": true,
+      "deleteTestResults": true
+    }
+  ],
+  "_links": {
+    "self": {
+      "href": "https://devdiv.visualstudio.com/DefaultCollection/0bdbc590-a062-4c3f-b0f6-9383f67865ee/_apis/build/Definitions/1054"
+    },
+    "web": {
+      "href": "https://devdiv.visualstudio.com/DefaultCollection/_permalink/_build/index?collectionId=011b8bdf-6d56-4f87-be0d-0092136884d9&projectId=0bdbc590-a062-4c3f-b0f6-9383f67865ee&definitionId=1054"
+    },
+    "editor": {
+      "href": "https://devdiv.visualstudio.com/DefaultCollection/_permalink/_build/definitionEditor?collectionId=011b8bdf-6d56-4f87-be0d-0092136884d9&projectId=0bdbc590-a062-4c3f-b0f6-9383f67865ee&definitionId=1054"
+    }
+  },
```

Comment:
I guess the same question applies for everything ignored in `IsSameContents` https://github.com/dotnet/buildtools/pull/1145/files#diff-376be9a04b43cb9d0a36ab900640771dR216, or maybe `VstsCreateDefinitionAsync`. It would be nice for this file to be minimal.